ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -147,19 +149,38 @@ export const getUpdatedProfileValue = (
   }
 };
 
-export const enableSave = (
+export const isBypassed = (
+  profile: Profile,
+  moduleName: AccessModule
+): boolean =>
+  !!getAccessModuleStatusByName(profile, moduleName)?.bypassEpochMillis;
+
+// would this AccessBypassRequest actually change the profile state?
+// allows un-toggling of bypass for a module
```

Comment:
does this comment make sense?  please help me make it more clear, if it does not